pkg/repositories/transformers/node_execution.go

```diff
@@ -88,6 +87,14 @@ func CreateNodeExecutionModel(input ToNodeExecutionModelInput) (*models.NodeExec
 		InputURI: input.Request.Event.InputUri,
 	}
 
+	// Start node is special case where the Inputs and Outputs are the same and hence here we copy the Output uri
+	// into input
+	if input.Request.Event.Id.NodeId == v1alpha1.StartNodeID {
+		// We dont need to check the result of type assertion here since for Startnode the putput result
+		// would always be OutputUri
+		nodeExecution.InputURI = input.Request.Event.OutputResult.(*event.NodeExecutionEvent_OutputUri).OutputUri
```

Comment:
what if the workflow has no inputs, and therefore the start node has no outputs? will the output uri be set? can we verify the output result exists before dereferencing it?

Reply:
I have removed this logic for now as we can have nodes with no Inputs.